custom_model_runner/datarobot_drum/drum/memory_monitor.py

```diff
@@ -35,6 +58,8 @@ def collect_memory_info(self):
             total=total_physical_mem_mb,
             avail=available_mem_mb,
             free=free_mem_mb,
-            predictor_rss=proc_rss_mb,
+            drum_rss=drum_rss_mb + nginx_rss_mb if drum_process else "drum process not found",
+            drum_info=drum_info if drum_process else "drum process not found",
```

Comment:
These should probably just pass None. Still not sure what it means if there is no drum.

Reply:
It builds memory `MemoryInfo` structure, which just shown in the `stats` request

Reply:
Okay but can you pass None though? why pass a string?

Reply:
To explain why it is missing.
Anyhow it can happen only when I'm debugging, and start not installed drum like:
PYTHONPATH=/path to drum/    source_code/drum

Reply:
If it only happens while you're debugging there's no reason to be verbose.

Reply:
fixed